tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -5607,6 +5607,67 @@ def test_create_event_source_validation(
         response = e.value.response
         snapshot.match("error", response)
 
+    @markers.aws.validated
+    def test_create_event_filter_criteria_validation(
+        self,
+        create_lambda_function,
+        lambda_su_role,
+        dynamodb_create_table,
```

Comment:
Does this test need a relatively heavy-weight DynamoDB stream or would a more lightweight SQS queue be sufficient here (could also simplify the test case by removing `StartingPosition`)?

Reply:
Tbh I just copied and adpated the above `test_create_event_source_validation` test. Was the quickest solution here. Since we are only ever doing a create (and not reading anything) I think the improvement to SQS will probably be very minimal.

The `StartingPosition` is required unfortunately for creating an ESM with dynamodb streams.